engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -89,10 +90,41 @@ def _init_model(self, kwargs, model_id_or_path):
             self._warmup(**kwargs)
 
     def _warmup(self, **kwargs):
-        batch_size = int(self.properties.get("max_rolling_batch_size", 32))
-        max_batch_prefill_tokens = int(
-            self.properties.get("max_rolling_batch_prefill_tokens", -1))
-        self.model.warmup(batch_size, max_batch_prefill_tokens)
+        max_prefill_tokens = int(
+            self.properties.get("max_rolling_batch_prefill_tokens", 4096))
```

Comment:
the purpose of `max_rolling_batch_prefill_tokens` has changed. Do we still want to keep this concept? And why we need now to introduce a max_prefill_tokens, can we just set some small value like 512, 1024?

Reply:
and why we start to bring this to external, can this be done inside lmi-dist?

Reply:
This was external before.

Reply:
why we set to 4096 such small value?  Does that have impact to the total token we can support? Or on the other hand, if customer set this value to be very high, like 32000, will this impact model loading?